docs_app/content/tutorial/1001-but-rxjs-is-hard.md

```diff
@@ -0,0 +1,108 @@
+# ???. But RxJS Is Hard
+
+We understand. Believe me. But maybe you're not seeing the real value of what RxJS offers, which is the [`Observable`](API) type itself. There are many [benefits to using observables](1000-why-observables.md), and you don't have to use [operators](GL) at all to get those benefits.
```

Comment:
Consider leaving off the "but maybe ..." sentence as it may be construed to be condescending.